MinPlatformPkg/Include/Library/PlatformConfigCheckLib.h

```diff
@@ -0,0 +1,26 @@
+/** @file
+
+  Platform configuration check library for TestPointCheckLib
+
+  Copyright (c) Microsoft Corporation. All rights reserved
+
+**/
+
+#ifndef _PLATFORM_CONFIG_CHECK_LIB_H_
+#define _PLATFORM_CONFIG_CHECK_LIB_H_
+
+#include <Uefi.h>
```

Comment:
I recommend removing this include from the header and just let it be handled in the including source file.

```suggestion
```